src/transform/transform.ts

```diff
@@ -100,9 +116,15 @@ async function process(session: Session<CodeModel>) {
   // fix up enum types
   for (const choice of values(session.model.schemas.choices)) {
     choice.choiceType.language.go!.name = schemaTypeToGoType(session.model, choice.choiceType, false);
+    // if (choice.language.go!.description) {
+    //   choice.language.go!.description = parseComments(choice.language.go!.description);
+    // }
```

Comment:
This is broken for compute, I'm investigating why the comments arent formatting correctly.